.gitignore

```diff
@@ -23,3 +23,18 @@ storybook-static
 .env
 wksps/
```

Comment:
Do we need to check-in the .yarn directory?

Reply:
yeah parts of it need to be checked in https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored